lit.go

```diff
@@ -39,8 +39,8 @@ type config struct { // define a struct for usage with go-flags
 	ReSync bool `short:"r" long:"reSync" description:"Resync from the given tip."`
 	Tower  bool `long:"tower" description:"Watchtower: Run a watching node"`
 	Hard   bool `short:"t" long:"hard" description:"Flag to set networks."`
-  
-	Verbose bool `short:"v" long:"verbose" description:"Set verbosity to true."`
+
+	LogLevel int `short:"v" long:"verbose" description:"Set verbosity level from 0 to 5 (most to least)"`
```

Comment:
This seems somewhat backwards, i'd expect the higher the number the more logging

Reply:
Cool, will change it in the next commit.